src/test/java/com/happeningnow/controller/ControllerOrganizerTest.java

```diff
@@ -0,0 +1,140 @@
+package com.happeningnow.controller;
+
+import com.happeningnow.model.Organizer;
+import com.happeningnow.repository.OrganizerRepository;
+import com.happeningnow.service.ServiceOrganizer;
+import org.junit.jupiter.api.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.reactive.WebFluxTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.reactive.server.WebTestClient;
+import reactor.core.publisher.Mono;
+import java.util.Collections;
+import java.util.Optional;
+import java.util.UUID;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.when;
+
+@WebFluxTest(controllers = OrganizerController.class)
+public class ControllerOrganizerTest {
+
+    @Autowired
+    private WebTestClient webTestClient;
+
+    @MockBean
+    private OrganizerRepository organizerRepository;
+
+    @MockBean
+    private ServiceOrganizer serviceOrganizer;
+
+    private Organizer organizer;
+
+    @BeforeEach
+    public void createOrganizer(){
+        UUID id = UUID.fromString("a0d3b612-cde9-417d-8c47-b268cc295e80");
+        organizer = new Organizer(
+                id,"Alex Sander","Developer","Portugal", Collections.emptyList()
+        );
+    }
+
+    @AfterEach
+    public void setUp(){
+        this.organizerRepository.deleteAll();
+    }
+
+    @Test
+    @DisplayName("This controller method should save an organizer")
+    public void save(){
+
+        when(serviceOrganizer.save(any(Organizer.class))).thenReturn(organizer);
+
+        webTestClient.post().uri("/happening-now/save-organizer")
+                .contentType(MediaType.APPLICATION_JSON)
+                .body(Mono.just(organizer), Organizer.class)
+                .exchange()
+                .expectStatus().isCreated()
+                .expectBody()
+                .jsonPath("$.id").isEqualTo(organizer.getId().toString())
+                .jsonPath("$.name").isEqualTo("Alex Sander")
+                .jsonPath("$.description").isEqualTo("Developer");
+    }
+
+    @Test
+    @DisplayName("This controller method should find an organizer by id")
+    public void findById(){
```

Comment:
this is meant to be an Integration test and as such, you should not use Mockito.

- Use a autowired version of the repo to persist one entry
- Call the web service to verify the data is there